.sauce/cypress.yml

```diff
@@ -21,14 +21,18 @@ docker:
 #  image:
 #    name: saucelabs/stt-cypress-mocha-node
 #    tag: v0.3.3
+
 cypress:
   version: 5.6.0
   configFile: "tests/e2e/cypress.json"  # We determine related files based on the location of the config file.
 suites:
   - name: "saucy test"
     browser: "chrome"
     platformName: "Windows 10"
+    screenResolution: "2560x1600"  # Available resolutions: '800x600', '1024x768', '1152x864', '1280x768', '1280x800', '1280x960', '1280x1024', '1400x1050', '1440x900', '1600x1200', '1680x1050', '1920x1080', '1920x1200', '2560x1600'
     config:
       env:
         hello: world
       testFiles: [ "**/*.*" ] # Cypress native glob support.
+      viewportHeight: 660
+      viewportWidth: 1000 # Cypress default resolution is 1000x660
```

Comment:
@tianfeng92 does these get passed to sauce-cypress-runner?

Reply:
@vrunoa yep. This will be passed to sauce-cypress-runner. I'll remove them in this PR and create another ticket to make it more clear.